eng/targets/ReferenceAssembly.targets

```
@@ -76,10 +77,15 @@
     </ItemGroup>
 
     <PropertyGroup>
+      <ManualReferenceAssemblyContent Condition="'$(ContainsManualReferenceAssemblyContent)' == 'true'">
```

```suggestion
    <_ManualReferenceAssemblyContent />
    <_ManualReferenceAssemblyContent Condition="Exists('$(_ManualRefSourceFileName)')">
```
Name change and empty default prevents all possible conflicts with other MSBuild goop.

Get rid of `$(ContainsManualReferenceAssemblyContent)` everywhere; one change should be sufficient to opt into including the .Manual.cs files.

---

Agreed, you should just probe for the file and if it exists include it.

---

I had this originally. Thought it felt excessively noisy and made this change hard to review. I can switch this back as a final commit if everything else is approved.